install.sh

```diff
@@ -64,11 +109,9 @@ do
 done
 sudo chown "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
 
-# Delete 'var' symlink on Cloudready platform
-if [[ $(grep neverware /etc/lsb-release) != "" ]]; then
-  [ -L /usr/local/var ] && sudo rm -f /usr/local/var
-  [ -L /usr/local/local ] && sudo rm -f /usr/local/local
-fi
+# Delete ${CREW_PREFIX}/{var,local} symlink on some Chromium OS distro if exist
+[ -L ${CREW_PREFIX}/var ] && sudo rm -f #{CREW_PREFIX}/var
+[ -L ${CREW_PREFIX}/local ] && sudo rm -f #{CREW_PREFIX}/local
```

Comment:
Probably should enclose in double quotes in case `#{CREW_PREFIX}` has a space.  Not that it ever should happen but it could potentially lead to the wrong files being deleted depending on the directory name.